tcbuilder/cli/bundle.py

```diff
@@ -51,7 +51,7 @@ def bundle(bundle_dir, compose_file, force=False, keep_double_dollar_sign=False,
 
     bundle_be.download_containers_by_compose_file(
         bundle_dir, compose_file, host_workdir,
-        output_filename=common.DOCKER_BUNDLE_FILENAME,
+        output_filename=f"{common.DOCKER_BUNDLE_TARNAME}.xz",
```

Comment:
Doesn't this setting make command "bundle" incompatible with the raw image? Or is this expected?

Reply:
Ok then.